src/styles/theme.ts

```diff
@@ -32,6 +32,54 @@ export const createSfTheme = (mode: "light" | "dark" = "light") =>
           disableRipple: true, // No more ripple, on the whole application!
         },
       },
+      MuiTable: {
+        styleOverrides: {
+          root: {
+            borderRadius: "7px",
+            border: `1px solid ${
+              mode === "light"
+                ? "rgba(224, 224, 224, 1)"
+                : "rgba(81, 81, 81, 1)"
+            }`,
+            borderCollapse: "initial",
+          },
+        },
+      },
+      MuiTableHead: {
+        styleOverrides: {
+          root: {
+            "&:hover .MuiTableSortLabel-root:not(.Mui-active):not(:hover) .MuiTableSortLabel-icon":
+              {
+                opacity: 0.5,
+              },
+          },
+        },
+      },
+      MuiTableBody: {
+        styleOverrides: {
+          root: {
+            "tr:last-child td": { border: 0 },
+          },
+        },
+      },
+      MuiTableCell: {
+        styleOverrides: {
+          body: {
+            padding: "0 16px",
+            height: "52px",
+          },
+          footer: {
+            padding: "0 8px",
+            height: "56px",
+            borderTop: `1px solid ${
+              mode === "light"
+                ? "rgba(224, 224, 224, 1)"
+                : "rgba(81, 81, 81, 1)"
+            }`,
+            borderBottom: "initial",
+          },
+        },
+      },
```

Comment:
Do you think some of these customizations should have comments or not?

Reply:
Good point. I will add comments to places that are harder to understand ;)

Reply:
Did you add any comments? :)